### PR TITLE
Peer Review

### PR DESCRIPTION
**Pull request by alain-kubwayo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,117 +4,202 @@
...
+                    <h3 class="title title--logo">Medical Core</h3>
+                    <ul class="header__list">
+                        <li class="header__list-item"><a class="header__link"href="#">Home</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Product</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Pricing</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="header__right">
+                    <a class="header__login-link" href="#">Login</a>
```
Comment (IbrahimBagalwa) [inline index.html:25]: Think the LOGIN link will have the same name with the lLEARN MORE btn. If you follow the figma link you can notice it.

File: index.html [hunk 2]
```diff
@@ -4,117 +4,202 @@
...
-            <form>
-                <input type="text" placeholder="Your Email">
-                <button class="cta-btn">Subscribe</button>
-            </form>
+        <div class="footer">
+            <div class="footer__container">
+                <div class="footer__desc">
+                    <h3 class="title title--small">Newsletter</h3>
+                    <h1 class="title title--big">JOIN US</h1>
+                    <p class="footer__text">Problems trying to resolve the conflict between <br>
```
Comment (IbrahimBagalwa) [inline index.html:194]: this footer__text should be like man__text but in footer text you add only the modifier because the footer text is center.

**Review by IbrahimBagalwa — COMMENTED** [review]
congratilations my dear! you have respected the rules of bem very well. keep it up. try to implament this comment i think it will be good. Thanks

File: index.css [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-gap: 40px;
+    margin-top: 80px;
+}
+
+.cards__card{
+    border-radius: 20px;
+    box-shadow: 0 0 30px rgba(0,0,0,0.18);
+
+}
```
Comment (NNesta) [inline index.css:213]: You should have created card block because it is a standalone and reusable component instead of making an element

**Review by NNesta — CHANGES_REQUESTED** [review]

**Review by NNesta — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+                <div class="main__desc">
+                    <h3 class="title title--small">Team</h3>
+                    <h2 class="title title--medium">Our Team</h2>
+                    <p class="main__text">
+                        Problems trying to resolve the conflict between <br> the two major realms of Classical 
+                        physics: Newtonian mechanics 
+                    </p>
+                </div>
+                <div class="cards">
+                    <div class="cards__card">
```
[the remark this answers is not in the document]
Comment (alain-kubwayo) [reply index.html:81]: Alright, working on that right away, thanks.

File: index.html (hunk 1 above)
Comment (alain-kubwayo) [reply index.html:25]: Oh okay, working on it.

**Review by NNesta — APPROVED** [review]
Nice work @alain-kubwayo

**Review by IbrahimBagalwa — APPROVED** [review]
Great job man keep it up :)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+                        <li class="header__list-item"><a class="header__link"href="#">Home</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Product</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Pricing</a></li>
+                        <li class="header__list-item"><a class="header__link" href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="header__right">
+                    <button class="btn btn--secondary btn--secondary-inactive">Login</button>
+                    <button class="btn btn--cta">
+                        <span>JOIN US</span>
```
Comment (Morrism1) [inline index.html:27]: `span` element with no class is not necessary in BEM convention.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+                        <img src="./assets/divider.png" alt="Horizontal Rule">
+                    </div>
+                    <h2 class="title title--medium">Meet Our Experts</h2>
+                    <p class="main__text">
+                        Problems trying to resolve the conflict between <br> 
+                        the two major realms of Classical physics: <br>
+                        Newtonian mechanics 
+                    </p>
+                    <button class="btn btn--main">
+                        <span>Learn More</span>
```
Comment (Morrism1) [inline index.html:67]: `span` element with no class is not necessary in BEM convention.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+                    <h3 class="title title--small">Team</h3>
+                    <h2 class="title title--medium">Our Team</h2>
+                    <p class="main__text">
+                        Problems trying to resolve the conflict between <br> the two major realms of Classical 
+                        physics: Newtonian mechanics 
+                    </p>
+                </div>
+                <div class="deck">
+                    <div class="card">
+                        <div>
```
Comment (Morrism1) [inline index.html:85]: `div` element with no class is not necessary in BEM convention.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+                                    </svg>    
+                                </a>
+                            </div> 
+                        </div>
+                    </div>
+                    <div class="card">
+                        <div>
```
Comment (Morrism1) [inline index.html:111]: `div` element with no class is not necessary in BEM convention.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
```
Comment (Morrism1) [inline index.html:137]: `div` element with no class is not necessary in BEM convention.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
```
Comment (Morrism1) [inline index.html:163]: `div` element with no class is not necessary in BEM convention.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+            </div>
+        </div>
+        <!-- End of Main -->
+        <!-- Footer -->
+        <div class="footer">
+            <div class="footer__container">
+                <div class="footer__desc">
+                    <h3 class="title title--small">Newsletter</h3>
+                    <h1 class="title title--big">JOIN US</h1>
+                    <p class="main__text">Problems trying to resolve the conflict between <br>
```
Comment (Morrism1) [inline index.html:198]: Here we have `main__text` which means I expect this element to be in a BLOCK called `main` but that's not the case.

**Review by Morrism1 — CHANGES_REQUESTED** [review]

**Review by IbrahimBagalwa — APPROVED** [review]
## Great job
keep it up bro you apply all request change :)

**Review by NNesta — APPROVED** [review]
Nice work @alain-kubwayo

**Review by Morrism1 — APPROVED** [review]
Good Job @alain-kubwayo!

**Pull request merged**